src/components/Mission.jsx

```diff
@@ -0,0 +1,42 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { Badge, Button, Container } from 'react-bootstrap';
+import Table from 'react-bootstrap/Table';
+import 'bootstrap/dist/css/bootstrap.css';
+
+import { fetchMissions, joinMission, leaveMission } from '../redux/mission/missionSlice';
+
+const Mission = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMissions());
+  }, [dispatch]);
+  const missions = useSelector((state) => state.missions);
+
+  return (
+    <Container className="mt-4 mb-4 bg-light">
+      <Table striped responsive bordered hover className="table">
+        <thead>
+          <tr>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th> </th>
+          </tr>
+        </thead>
+        <tbody>
+          {missions.missions.map((item) => (
+            <tr key={item.id}>
+              <td>{item.name}</td>
+              <td>{item.desc}</td>
+              <td><Badge bg={!item.reserved ? 'secondary' : 'success'}>{!item.reserved ? 'NOT A MEMBER' : 'Active Member'}</Badge></td>
+              <td><Button as="a" className={!item.reserved ? 'bg-primary' : 'bg-danger'} onClick={() => (item.reserved ? dispatch(joinMission(item.id)) : dispatch(leaveMission(item.id)))}>{!item.reserved ? 'Join Mission' : 'Leave Mission'}</Button></td>
+            </tr>
+          ))}
+        </tbody>
+      </Table>
```

Comment:
- [ ] @Elhambasir and @OnlyUnknown Great work on your mission page, however, some design aspects don't match the design given eg the join and leave the mission, active member not a member, also consider increasing the width of the table and reducing font size of the description section 

- [ ] It would be nice reducing the size of join and leave buttons they are too big

Good Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Elhambasir/group-project-space-travelers-hub/assets/61408860/58271fd2-8e2e-4e4d-8dcf-e10e64472cdf)  | ![image](https://github.com/Elhambasir/group-project-space-travelers-hub/assets/61408860/9ddb351f-ed98-404c-a6f9-ed0cb10a0411)